web/turkey/survey/views.py

```diff
@@ -1,106 +1,215 @@
 from django.apps import apps
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.staticfiles.templatetags.staticfiles import static
 from django.core.paginator import Paginator
+from django.core.urlresolvers import reverse
 from django.db import transaction
-from django.http import Http404, StreamingHttpResponse, HttpResponse
+from django.http import Http404, StreamingHttpResponse
 from django.template.response import TemplateResponse
-from django.utils.decorators import method_decorator
+from django.utils.safestring import mark_safe
 from django.utils.translation import ugettext_lazy as _
-from django.views.decorators.csrf import ensure_csrf_cookie
 from django.views.generic import View
 from rest_framework import status
+from rest_framework.authentication import BaseAuthentication
 from rest_framework.exceptions import ValidationError
+from rest_framework.permissions import AllowAny
 from rest_framework.views import APIView
 from rest_framework.response import Response
 
 from .auditors import NAME_TO_AUDITOR
 from .steps import NAME_TO_STEP
 from .models import Task, TaskInteraction
 
-
 # TODO: Authentication
-from survey.renderers import XMLBodyRenderer
+from .renderers import XMLBodyRenderer
+
+
+# TODO: Implement token authentication. So when the interaction is created, make a token to identify user...
```

Comment:
We need a tracking and authentication system or we're wide open and we can't track a single user